main.tex

```diff
@@ -337,7 +337,7 @@ \section{Hackathon Details and Setup}
 \bottomrule
 \endlastfoot
 
-\input{|python3 python_scripts/process_spreadsheet.py}
+\input{projects_table.tex}
```

Comment:
You're supposed to leave it with running the Python scripts (mostly because that spreadsheet comes from a Google doc), but it looks like you hard-coded this instead. Could you explain? Did you autogenerate this by running the Python script?